src/main/resources/RustIconMappings.json

```diff
@@ -0,0 +1,49 @@
+{
+    "icons": {
+        "expui": {
+            "rust.svg": "icons/rust.svg",
+            "rustBuild.svg": "icons/rustBuild.svg",
+            "rustReload.svg": "icons/rustReload.svg",
+            "rustRepl.svg": "icons/rustRepl.svg",
+            "toolWindow.svg": "icons/toolWindow.svg",
+
+            "rustFile.svg": "icons/rustFile.svg",
+            "rustMain.svg": "icons/rustMain.svg",
+            "rustMod.svg": "icons/rustMod.svg",
```

Comment:
Do we really need to duplicate icons in this case?

Reply:
I don't see other ways to do it.

Reply:
👌